eth/protocols/eth/handlers.go

```diff
@@ -240,6 +240,41 @@ func handleGetBlock66(backend Backend, msg Decoder, peer *Peer) error {
 	return nil
 }
 
+func handlePendingEtxs(backend Backend, msg Decoder, peer *Peer) error {
+	// Decode the block pending etxs retrieval message
+	ann := new(PendingEtxsPacket)
+	if err := msg.Decode(&ann); err != nil {
+		return fmt.Errorf("%w: message %v: %v", errDecode, msg, err)
+	}
+	// Mark the hashes as present at the remote node
+	peer.markPendingEtxs(ann.PendingEtxs.Header.Hash())
+
+	return backend.Handle(peer, ann)
+}
+
+func handleGetOnePendingEtxs(backend Backend, msg Decoder, peer *Peer) error {
+	// Decode the block pending etxs retrieval message
+	var query GetOnePendingEtxsPacket
+	if err := msg.Decode(&query); err != nil {
+		return fmt.Errorf("%w: message %v: %v", errDecode, msg, err)
+	}
+	pendingEtxs := backend.Core().GetPendingEtxs(query.Hash)
+	log.Trace("Completing  a pendingEtxs request for", "Hash", pendingEtxs.Header.Hash())
+	return peer.SendPendingEtxs(*pendingEtxs)
+}
+
+func handleGetOnePendingEtxs66(backend Backend, msg Decoder, peer *Peer) error {
```

Comment:
Are we making two versions of the same functions to handle a deprecated encoding scheme?

Reply:
removed eth65